docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# Notes Metrics
+The metrics collection and analysis plan for Notes, a forthcoming Test Pilot experiment.
+
+
+## Analysis
+Data collected by Notes will be used to answer the following high-level questions:
+
+- Do people take notes in Firefox?
+	- In what intervals?
+	- In what circumstances?
+- What are the notes like?
+	- How long are they?
+	- Do they span multiple lines?
+	- Do they use formatting options? Which?
+- Do people sync?
+	- How does syncing affect engagement?
+
+Note: sync will not be part of the first release of this experiment; all sync-related metrics will be reported when that feature is offered.
+
+## Collection
+Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
+
+### Custom Metrics
+- `cm1` - the size of the notepad, in characters, with markup removed.
```

Comment:
Unfortunately, we're stuck with those names; that's just how GA's custom dimensions and metrics work. Some links:

- https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md#custom-dimensionmetric-fields
- https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#customs